charts/lineCharts/LineChart.tsx

```diff
@@ -225,30 +216,28 @@ class Lines extends React.Component<LinesProps> {
         ))
     }
 
-    container?: SVGElement
+    private container?: SVGElement
     componentDidMount() {
         const base = this.base.current as SVGGElement
-        this.container = base.closest("svg") as SVGElement
-
-        this.container.addEventListener("mousemove", this.onCursorMove)
-        this.container.addEventListener("mouseleave", this.onCursorLeave)
-        this.container.addEventListener("touchstart", this.onCursorMove)
-        this.container.addEventListener("touchmove", this.onCursorMove)
-        this.container.addEventListener("touchend", this.onCursorLeave)
-        this.container.addEventListener("touchcancel", this.onCursorLeave)
+        const container = base.closest("svg") as SVGElement
```

Comment:
We've got another global here and should probably clean that up in a later commit.